### ISSUE TITLE
Legacy Webapp: assets.py attempts to add epithets via the legacy app when adding Fighting Arts

### ISSUE THREAD

**Issue by toconnell** [toconnell/kdm-manager#268]
This is causing tracebacks.

From email:

```
[5984768b8740d928e0e32e5b] was logged out of the webapp instance on ubuntu-trusty-1 due to a render failure at 2017-08-04 22:25:36.783726.

 The traceback from the exception was this:
Traceback (most recent call last):
File "/home/toconnell/kdm-manager/v1/session.py", line 63, in wrapper
 return func(self, *args, **kwargs)
File "/home/toconnell/kdm-manager/v1/session.py", line 678, in process_params
 S.modify(self.params)
File "/home/toconnell/kdm-manager/v1/assets.py", line 1813, in modify
 self.add_game_asset("fighting_art", game_asset_key)
File "/home/toconnell/kdm-manager/v1/assets.py", line 1103, in add_game_asset
 self.update_epithets(epithet=asset_dict["epithet"])
AttributeError: Survivor instance has no attribute 'update_epithets'
```

Here is the offending code:
```
1054     def add_game_asset(self, asset_type, asset_key, asset_desc=None):
1055         """ Our generic function for adding a game_asset to a survivor. Some biz
1056         logic/game rules happen here.
1057
1058         The kwarg 'asset_type' should always be the self.name value of the
1059         game_asset model (see models.py for more details).
1060
1061         Finally, if, for whatever reason, we can't add the asset, we return
1062         False.
1063         """
...
1095         elif asset_type == "fighting_art":
1096             if asset_key == "RANDOM_FIGHTING_ART":
1097                 fa_deck = FightingArts.build_survivor_deck(self, self.Settlement)
1098                 self.update_fighting_arts(random.choice(fa_deck), action="add")
1099             elif asset_key in FightingArts.get_keys():
1100                 self.survivor["fighting_arts"].append(asset_key)
1101                 asset_dict = FightingArts.get_asset(asset_key)
1102                 if "epithet" in asset_dict.keys():
1103                     self.update_epithets(epithet=asset_dict["epithet"])
1104                 if "Movement" in asset_dict.keys():
1105                     self.survivor["Movement"] = int(self.survivor["Movement"]) + asset_dict["Movement"]
1106             else:
1107                 self.logger.exception("[%s] Attempted to add unknown fighting art!" % self)
1108                 return False
```

Honestly, we should probably just move this whole method to the API, at this point, since they'll need it for the alpa Survivor Sheet.

**Comment by toconnell** [comment]
Hat-tip goes to _Caffran2000_

**Comment by toconnell** [comment]
Related to issue #81, weirdly

**Comment by toconnell** [comment]
This was fixed in 2.26.608
